testing/cpm/cpm_libcudacxx-export.cmake

```diff
@@ -21,6 +21,12 @@ rapids_cpm_init()
 rapids_cpm_libcudacxx(BUILD_EXPORT_SET frank INSTALL_EXPORT_SET test)
 rapids_cpm_libcudacxx(INSTALL_EXPORT_SET test2)
 
+add_library(example INTERFACE )
```

Comment:
```suggestion
add_library(example INTERFACE)
```

Reply:
I believe that we can't change this as it will fail the style checks

Reply:
Hmm, I didn’t expect that. Is the whitespace meaningful in the syntax?

Reply:
No. But cmake-format likes to remove white space before ( and )